website/static/js/atjsConfig.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * At.js config.
+ */
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+
+require('Caret.js');
+require('At.js');
+
+// preventing unexpected contenteditable behavior
+
+// prevent pasting HTML
+var onPaste = function(e) {
```

Comment:
If there is any code copied from somewhere else, it's very helpful to provide the source as it helps us distinguish copied code from custom code. Also provide any attribution if necessary.

Not a blocker, as long as we're following the terms of this code's license.